tests/conftest.py

```
@@ -2629,6 +2629,11 @@ def rwx_fs_available_storage_classes_names(cluster_storage_classes_names):
     ]
 
 
+@pytest.fixture()
```

Ruff (0.15.14)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)